opentelemetry-collector/manifest.yaml

```diff
@@ -2,61 +2,66 @@ dist:
\ No newline at end of file
+  - github.com/open-telemetry/opentelemetry-collector-contrib/internal/common => github.com/open-telemetry/opentelemetry-collector-contrib/internal/common 9b0e90027913f53d4231eb1762063cf67df61051
+  - github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal => github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal 9b0e90027913f53d4231eb1762063cf67df61051
+
+  - github.com/open-telemetry/opentelemetry-collector-contrib/exporter/kafkaexporter => github.com/open-telemetry/opentelemetry-collector-contrib/exporter/kafkaexporter 9b0e90027913f53d4231eb1762063cf67df61051
+  - github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusremotewriteexporter => github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusremotewriteexporter 9b0e90027913f53d4231eb1762063cf67df61051
+  - github.com/open-telemetry/opentelemetry-collector-contrib/exporter/zipkinexporter => github.com/open-telemetry/opentelemetry-collector-contrib/exporter/zipkinexporter 9b0e90027913f53d4231eb1762063cf67df61051
+
+  - github.com/open-telemetry/opentelemetry-collector-contrib/pkg/resourcetotelemetry => github.com/open-telemetry/opentelemetry-collector-contrib/pkg/resourcetotelemetry 9b0e90027913f53d4231eb1762063cf67df61051
+  - github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/jaeger => github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/jaeger 9b0e90027913f53d4231eb1762063cf67df61051
+  - github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/opencensus => github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/opencensus 9b0e90027913f53d4231eb1762063cf67df61051
+  - github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/zipkin => github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/zipkin 9b0e90027913f53d4231eb1762063cf67df61051
+
+  - github.com/open-telemetry/opentelemetry-collector-contrib/receiver/opencensusreceiver => github.com/open-telemetry/opentelemetry-collector-contrib/receiver/opencensusreceiver 9b0e90027913f53d4231eb1762063cf67df61051
+  - github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver => github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver 9b0e90027913f53d4231eb1762063cf67df61051
+  - github.com/open-telemetry/opentelemetry-collector-contrib/receiver/scraperhelper => github.com/open-telemetry/opentelemetry-collector-contrib/receiver/scraperhelper 9b0e90027913f53d4231eb1762063cf67df61051
+  - github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver => github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver 9b0e90027913f53d4231eb1762063cf67df61051
```

Comment:
I believe these will be able to be removed once `v0.34.0` is tagged as they will reference a valid version instead of the fake pseudoversion.

Reply:
Sure, but then they will not work on unreleased versions, so I prefer to keep them :)